service/controller/v13/version_bundle.go

```diff
@@ -52,6 +52,26 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Set version bundle version stack output via parameter.",
 				Kind:        versionbundle.KindAdded,
 			},
+			{
+				Component:   "cloudconfig",
```

Comment:
I was thinking to have constants for the components so everybody can check which ones are available and we get them right reliably. Just not sure where to put them.

Reply:
I like that but where to put them is tricky. They vary by provider but having them in a single place is easier to review / understand IMO.

What about `versionbundle` it also has the component data structure?

Reply:
Having common component constants in there sounds good to me.